packages/authorization-process/src/services/authorizationService.ts

```
@@ -1341,6 +1344,32 @@ export function authorizationServiceBuilder(
         )
       );
     },
+    async removeClientAdmin(
+      { clientId, adminId }: { clientId: ClientId; adminId: UserId },
+      { correlationId, logger, authData }: WithLogger<AppContext<UIAuthData>>
+    ): Promise<void> {
+      assertFeatureFlagEnabled(config, "featureFlagAdminClient");
```

If I understand correctly, the `assertFeatureFlagEnabled` throws an error in case the feature flag is not enabled. However this error is not handled in the error mapper, so it will result in a `500`. Is this a desired behavior?
cc @sandrotaje

---

Yes, if the feature flag is not enabled it should go into error and for the moment it returns a 500, I don't know if it will be mapped later or with the PR of @sandrotaje

---

In #1856 Sandro added a default mapping that maps this error to 403 by default, in case it's not mapped to any other code by the errorMapper passed as param